airbyte-db/db-lib/src/main/java/io/airbyte/db/JdbcCompatibleSourceOperations.java

```diff
@@ -20,41 +18,21 @@
    *
    * @param colIndex 1-based column index.
    */
-  void setJsonField(final ResultSet resultSet, final int colIndex, final ObjectNode json) throws SQLException;
+  void putJsonField(final ResultSet resultSet, final int colIndex, final ObjectNode json) throws SQLException;
```

Comment:
What's the rationale behind `put` vs `set`? Is it primarily because most of the `AbstractJdbcCompatibleSourceOperations` uses `put{DataType}` in their method naming? It seems like `put` is typically used for updating a value, however, based on the Javadoc comment it seems that we're copying.

Reply:
Yes, primarily for that. But renaming to `copyToJsonField` - thanks for the suggestion